### DISCUSSION TITLE
Pushover erhält nach Update keine Meldungen mehr

### DISCUSSION THREAD

**Discussion by Kaki84** [OpenCCU/OpenCCU#2586] [🇩🇪 Fragen&Antworten] [answered]
Hallo zusammen, ich troubleshoote schon super lange, aber finde einfach keinen Fehler:
Ich habe das neuste Update der Software installiert und seit dem kommen keine Push-Nachrichten mehr an. Also gehe ich davon aus das das Skript nicht korrekt ausgeführt wird. Aber ich finde einfach keinen Anhaltspunkt, was ich machen kann. Jemand eine Idee? Habe auch das Forum durchsucht aber nichts passendes gefunden.
Danke euch vielmals!!! 

Das Skript läuft seit Jahren und sieht so aus:

string po_api_user="*************************************";
string po_api_token="*************************************";

string po_title="Alarmausl%C3%B6sung";
string po_message="Bewegungsmelder Windfang hat Bewegung erkannt!";
string po_sound="none";
string po_priority="1";

string po_retry="30";
string po_expires="3600";

string po_request;
if(po_priority=="2"){
po_request="/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=" # po_api_token # "&user=" # po_api_user # "&title=" # po_title # "&priority="# po_priority # "&sound=" # po_sound # "&retry="# po_retry # "&expire=" # po_expires # "&message=" # po_message # "\" https://api.pushover.net/1/messages.json";
}
else{
po_request="/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=" # po_api_token # "&user=" # po_api_user # "&title=" # po_title # "&priority="# po_priority # "&sound=" # po_sound # "&message=" # po_message # "\" https://api.pushover.net/1/messages.json";
}
dom.GetObject("CUxD.CUX2801001:1.CMD_EXEC").State(po_request);

**Comment by blackbasket** [comment]
Guten Morgen @Kaki84 ,

was mir auf den ersten Blick auffällt, sind die unterschiedlichen Anführungszeichen in den ersten beiden Zeilen...

LG,
Marcel

**Reply by Kaki84** [reply]
Hi, danke, ja da stehen eigentlich die beiden Infos zum Account, die habe ich gelöscht, in dem Auszug ;) dadurch die falsche Darstellung 

![20240105_093203.jpg](https://github.com/jens-maus/RaspberryMatic/assets/155715625/2268701c-b90a-4248-92e4-66afa6620fa0)

**Comment by Kaki84** [comment]
Ich komme näher ran - es hat irgendwas mit dem Aufruf zu tun. Offensichtlich ist da was anders im CUx Deamon. Wer was weiß... ;-)

**Comment by Kaki84** [comment]
Nun komme ich deutlich Näher!

Das hier ist der Aufruf im Skript:

string po_request;
if(po_priority=="2"){
po_request="/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=" # po_api_token # "&user=" # po_api_user # "&title=" # po_title # "&priority="# po_priority # "&sound=" # po_sound # "&retry="# po_retry # "&expire=" # po_expires # "&message=" # po_message # "\" https://api.pushover.net/1/messages.json";
}
else{
po_request="/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=" # po_api_token # "&user=" # po_api_user # "&title=" # po_title # "&priority="# po_priority # "&sound=" # po_sound # "&message=" # po_message # "\" https://api.pushover.net/1/messages.json";
}
dom.GetObject("CUxD.CUX2801001:1.CMD_EXEC").State(po_request);

Das hier ist die Fehlermeldung: (Accountdaten geXXXXed):

Jan  5 10:00:47 homematic-raspi local0.warn ReGaHss: WARNING: XMLRPC 'setValue': empty interface URL (interface: 1325, params: {"CUX2801001:1","CMD_EXEC","/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=XXXXXXXX&user=XXXXXXXX&title=Alarmausl%C3%B6sung&priority=1&sound=none&message=Bewegungsmelder Windfang hat Bewegung erkannt!\" https://api.pushover.net/1/messages.json"}) [CallXmlrpcMethod():iseXmlRpc.cpp:
Jan  5 10:00:47 homematic-raspi local0.err ReGaHss: ERROR: XMLRPC 'setValue' call failed (interface: 1325, params: {"CUX2801001:1","CMD_EXEC","/usr/local/addons/cuxd/curl -X POST -k -H \"Content-Type: application/x-www-form-urlencoded\" -d \"token=XXXXXXXXXXX=XXXXXXXX&title=Alarmausl%C3%B6sung&priority=1&sound=none&message=Bewegungsmelder Windfang hat Bewegung erkannt!\" https://api.pushover.net/1/messages.json"}) [CallSetValue():iseXmlRpc.cpp:1515]

**Reply by Kaki84** [reply]
Und nu? :-) hehe

**Reply by blackbasket** [reply]
Wo siehst du die Fehlermeldung? Und welche CUx-Version?

**Reply by Kaki84** [reply]
Danke für deine Unterstützung!
Die Fehlermeldung sehe ich im CUx Deamon, neuste Version, siehe Screenshot.
Ich  finde einfach den Fehler nicht....

![20240105_115651~2.jpg](https://github.com/jens-maus/RaspberryMatic/assets/155715625/47e6f592-7fcd-45b4-8ae8-e401684aa87a)

**Reply by blackbasket** [reply]
Also in meiner Erfolgsmeldung wird die `url: xmlrpc_bin://127.0.0.1:8701` angezeigt... hast du schon mal einen anderen Channel vom CUxD-Device versucht?

**Reply by Kaki84** [reply]
Wow - okay. Danke dir soweit für den Support. Ich habe gerade aus lauter Verzweifellung das Gerät nochmals neu gebootet - und Zack: Alle Fehler sind behoben. Keine Ahnung warum oder wieso. Ich danke dir trotzdem für die Unterstützung und es tut mir leid das ich kein befriedigendes Ergebnis habe warum das Verhalten aufgetreten ist.

**Reply by blackbasket** [reply, marked as answer]
Reboot tut gut ;)

Gerne
